test/main.test.js

```diff
@@ -215,10 +215,11 @@ describe("sqlparser", function () {
         "select `t1`.id, t3.* from t1, t2 join t3 join (select * from t4);",
         { t1: "table1", t2: "table2", t3: "table3" } // Leave t4 "as is"
       );
+
       // Note the canonical "join" added below to replace the comma
       strictEqual(
         statements.join(""),
-        "select table1.id,table3.* from table1 join table2 join table3 join(select * from t4)"
+        "select `table1`.id,table3.* from table1 join table2 join table3 join(select * from t4)"
       );
       deepStrictEqual(tables, ["table1", "table2", "table3", "t4"]);
```

Comment:
Now the return value from normalize leaves the escapes in the statement string, but the table names do not include the escape chars.